package.json

```diff
@@ -21,15 +21,18 @@
     "tailwindcss": "2.0.2"
   },
   "devDependencies": {
+    "@types/classnames": "2.2.6",
```

Comment:
Could you update Next.js too? I noticed because the generated types file and tsconfig will be slightly different.

Reply:
I've updated Next to v12, but do let me know if you had another version in mind, I didn't see any breaking changes :smile: